Cargo.toml

```diff
@@ -29,6 +29,7 @@ indexmap = "^1"
 
 [dev-dependencies]
 doc-comment = "^0.3"
+serial_test = "0.9.0"
```

Comment:
this is not needed. cargo-make already supports it built in.
just add 'multi-phase-tests' to the profile list in the project makefile and add ignore macro for tests that need to run in single thread.
cargo make will first run all parallel tests and than only 'ignored' single thread tests